scipy/optimize/__init__.py

```diff
@@ -255,12 +255,21 @@
 
    linprog_verbose_callback -- Sample callback function for linprog (simplex).
 
-Assignment problems:
+Assignment problems
+===================
 
 .. autosummary::
    :toctree: generated/
 
    linear_sum_assignment -- Solves the linear-sum assignment problem.
+   quadratic_assignment -- Approximates solution to the quadratic assignment problem and the graph matching problem.
```

Comment:
This line is too long. Is anyone sure of a way that we can break the line without breaking the appearance of the rendered documentation? I've tried a few things but can't seem to get it right, and I can't iterate rapidly because I can't build docs locally.

Reply:
Does it fit if you delete the part about graph matching?

Reply:
Yes, but I think that's a good thing to include, no? Those not really familiar with the problem might not know that the algorithms often work for both.

Reply:
It brings me back to my comment about QAP encompassing TSP, graph isomorphism etc.
`linear_sum_assignment` also has many diverse applications but we don't mention them here.

Reply:
@asaadeldin11 @jovo Could you weigh in on this?

Reply:
graph isomorphism is a subset of graph matching, falling under exact graph matching. TSP is a problem that be solved via the QAP. IMO, the Graph Matching Problem and the QAP are two broad problems, with objective functions that are the negation of the other, and many applications. I would not say the Graph Matching Problem is an application of QAP. It is a separate problem with a formulation that is the negation of that of QAP, so any algorithm designed to solve the QAP can also be used on the GMP.

Reply:
However, I don't think it would be a bad to mention the TSP in the documentation, since it is a common application of QAP.